### ISSUE TITLE
[Bug Report] FTU App - Web Extensions - Keyboard only access for toggles

### ISSUE THREAD

**Issue by tylerchilds** [capyloon/nutria#67]
**Describe the bug**
Currently the web extensions cannot be marked with keyboard only input.

Actual:

** Environment **
- Rasbperry Pi 3B+

**To Reproduce**
1. Open FTU App
2. Progress to Web Extensions
3. Tabbing focus to either the item cell or the button in the item cell
4. Press enter or space bar, but nothing happens

**Expected behavior**
The currently selected extension should toggle on or off

**Additional Context**
Testing in preparation for Raspberry Pi 400 support.
